compute_funcx/endpoint/README.md

```diff
@@ -0,0 +1,6 @@
+# funcx_endpoint Wrapper Package
+
+* This package serves as a backwards compatible library for current funcx-endpoint users who may upgrade to the latest code but
+does not want to make any changes to existing scripts that references funcx_endpoint classes to use globus_compute_endpoint.* instead.
+
+* Accomplished via a variety of import globus_compute_endpoint.compute_class_X as class_X in __init__.py and other files
```

Comment:
As this is markdown, you might consider adding a couple of details like backticks:

```
... for current `funcx-endpoint` users ...

... `import globus_compute_endpoint.com...

... in `__init__.py` and ...
```

Reply:
Used backticks.